addons/hw_drivers/controllers/driver.py

```diff
@@ -102,9 +109,15 @@ def connect_box(self, token):
         image = get_resource_path('hw_drivers', 'static/img', 'False.jpg')
         if server == '':
             token = b64decode(token).decode('utf-8')
-            url, token = token.split('|')
+            try:
+                # IoT Box send token with db_uuid and enterprise_code only since V13
+                url, token, db_uuid, enterprise_code = token.split('|')
```

Comment:
ca marche pas non plus ici, parce que tu dois être rétrocompatible avec les gens en 12.0 qui n'auront qu'un seul "|".

et ca ferra ca: alueError: not enough values to unpack (expected 4, got 2)